urizen/generators/locations/lgd_grid.py

```diff
@@ -0,0 +1,95 @@
+import random
+from urizen.core.map import Map
+from urizen.core.cell_collection import (
+    cell_dungeon_wall,
+    cell_dungeon_floor,
+    cell_dungeon_closed_door,
+    cell_void
+)
+
+
+def lgd_grid_simple(w, h, room_size=4):
+    M = Map(w, h, fill_cell=cell_dungeon_floor)
+    _create_room_grid(M, room_size=room_size)
+    _create_door(M, room_size=room_size)
+    _crush_walls(M, room_size=room_size)
+    _delete_walls(M)
+    return M
+
+
+def _create_room_grid(M, room_size):
+    w, h = M.get_size()
+    x_border_index = ((w - 1) // (room_size + 1)) * (room_size + 1)
+    y_border_index = ((h - 1) // (room_size + 1)) * (room_size + 1)
+    for y, line in enumerate(M.cells):
+        for x, C in enumerate(line):
+            if x > x_border_index or y > y_border_index:
+                M.cells[y][x] = cell_void(x, y)
+            elif x % (room_size + 1) == 0 or y % (room_size + 1) == 0:
+                M.cells[y][x] = cell_dungeon_wall(x, y)
+
+def _create_door(M, room_size):
```

Comment:
This function creates many doors so the `_create_doors` name is better

+docstring